### ISSUE TITLE
App doesn't start after the upgrade

### ISSUE THREAD

**Issue by oguzgundogdu** [dbeaver/dbeaver#35694]
### Description

After the upgrade from 24.2 to 24.2.1, the app crashes during the start
![image](https://github.com/user-attachments/assets/da4868df-8cb8-47ec-9241-1394e83a8bfd)

[1727163969816.log](https://github.com/user-attachments/files/17110327/1727163969816.log)

### DBeaver Version

24.2.1

### Operating System

Windows Server 2016 Data Center

### Database and driver

MS SQL Server

### Steps to reproduce

Start the app

**Comment by LowkeyLieSmith** [comment]
Same issue today on Windows Server 2016

**Comment by Matvey16** [comment]
@oguzgundogdu Hello, how did you install dbeaver? And what exactly version of WIndows Server 2016 do you use. You can find it if you execute `winver`

**Comment by carlosesilvadev** [comment]
@Matvey16  Oi, então eu instalei usando o link Windows (zip) https://dbeaver.io/files/dbeaver-ce-latest-win32.win32.x86_64.zip, depois de extraído na máquina foi realizado a tentativa de abrir o dbeaver.exe e é apresentado o erro indicado pelo @oguzgundogdu. A versão do S.O. usando o winver é Windows Server 2016 14393
<img width="341" alt="image" src="https://github.com/user-attachments/assets/7a749a31-c25d-4d52-ba66-0da5263b941f">

**Comment by oguzgundogdu** [comment]
@Matvey16 Hi! I just upgraded as I always do through the upgrade dialog. Then I tried to make a clean install by uninstalling and installing again.
Here below, my version. It's very close to @avlisesolrac's one
![image](https://github.com/user-attachments/assets/a1b601ba-2abf-4d3d-a670-f0179462ebbd)

Thanks

**Comment by Phillipus** [comment]
Windows Server 2016 is no longer supported by Eclipse SWT. See https://github.com/eclipse-platform/eclipse.platform.swt/issues/1500

**Comment by E1izabeth** [comment]
The issue comes from the platform on which DBeaver is based.
Here are 2 options that might help:
- Downgrade DBeaver to 24.2.0
- Try upgrading Windows. If no additional minor upgrades are available, upgrade to a higher major release of Windows.

**Comment by 2019-05-10** [comment]
> Windows Server 2016 is no longer supported by Eclipse SWT. See [eclipse-platform/eclipse.platform.swt#1500](https://github.com/eclipse-platform/eclipse.platform.swt/issues/1500)

Well, maybe some sort of warning before/instead of the "update available" info would have been useful here.

**Comment by E1izabeth** [comment]
Yes, we're going to implement it. Here is the issue about it #34315
